definitions/routing.lua

```diff
@@ -0,0 +1,249 @@
+-- move all of these down into rc config
+--
+-- or should these be put in routing lib?
+local TRACK_INFO_AUDIO_SRC_DISABLED = -1
+local TRACK_INFO_MIDIFLAGS_ALL_CH = 0
+local TRACK_INFO_MIDIFLAGS_DISABLED = 4177951
+local TRACK_INFO_CATEGORY_SEND = 0 -- send
+local TRACK_INFO_CATEGORY_RECIEVE = -1 -- send
+local TRACK_INFO_CATEGORY_HARDWARE = 1 -- send
+
+--//////////////////////////////////////////////////
+--
+-- ROUTING
+--
+--  neither of a and m | a and not m
```

Comment:
there are many comments here as well. they are difficult to understand for me too. Another reason why comments are evil in general is that they tend to duplicate information, and can grow out of data when code changes, leading to great  confusion

Reply:
i have to clean up that file. it is way to messy!